# Experiment 211d63a091ab4b6dffe24c5bb2ee0c041a0066b5399f4b8b31b3f62f9a315947
# Base 07ff7cbd8bef0dfa5407c1637452c95f7b41f2f2
diff --git a/src/m_co_obj.cpp b/src/m_co_obj.cpp
index a6169c753..e7f54d212 100644
--- a/src/m_co_obj.cpp
+++ b/src/m_co_obj.cpp
@@ -477,6 +477,9 @@ void RGE_Master_Combat_Object::modify_percent(float value,
                                                 unsigned char attribute)
 {
     switch (attribute) {
+    case 8:
+    case 9:
+        break;
     case 6:
         turn_speed = (float)((double)value * (double)turn_speed);
         break;
